src/sdk/pynni/nni/tuner.py

```diff
@@ -98,6 +98,13 @@ def save_checkpoint(self):
         checkpoin_path = self.get_checkpoint_path()
         _logger.info('Save checkpoint ignored by tuner, checkpoint path: %s' % checkpoin_path)
 
+    def feed_tuning_data(self, data):
+        """Feed additional data for tuning
+        data: a list of dictionarys, each of which has at least two keys, 'parameter' and 'value'
+        """
+        _logger.info('Feeding...')
+        pass
```

Comment:
If you pass the function, why you use "feeding..." log info? (In fact, you are not feeding...)

Reply:
Have removed pass